### PR TITLE
Rename language option from "phone language" to "device language"

### LINKED ISSUE #1131: Rename language option from "phone language" to "device language"
General Settings > User interface > Language -- the default option is "Use phone language" but not everyone is on a phone so it should be "use device language".

### PR DESCRIPTION
**Pull request by cdwuestner**
PR to resolve #1131

### CODE REVIEW

**Comment by lognaturel** [comment]
Thank you, @cdwuestner and welcome to ODK!

There's an added little twist to this one -- we have a lint check to make sure that there are no unused resources which is currently failing. You can run it locally with `./gradlew lint` or get the analysis by clicking on "Details" next to the circleci (continuous integration) results. It should be a quick fix and then we'll be ready to merge!

**Comment by shobhitagarwal1612** [comment]
@lognaturel @cdwuestner I feel that instead of removing the existing string `use_phone_language` we should just change the value of that string from `Use device language` to `Use phone language`

**Comment by shobhitagarwal1612** [comment]
@cdwuestner I guess the test is failing because of the presence of unused string resources. 
To remove them quickly you can follow these steps.

1. Goto res > values in the Project tree structure located on the left side of Android Studio
2. Right click on `strings` folder
3. Choose `refactor` > `Remove unused resources`

**Comment by cdwuestner** [comment]
Sorry about that! Thanks for the suggestion, @shobhitagarwal1612

File: collect_app/src/main/res/xml/analytics_tracker.xml [hunk 1]
```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (shobhitagarwal1612) [inline collect_app/src/main/res/xml/analytics_tracker.xml]: please add this file back

Comment (lognaturel) [reply collect_app/src/main/res/xml/analytics_tracker.xml]: Thanks, @shobhitagarwal1612! @cdwuestner I believe this file is implicitly used by the analytics library. The Android Studio tools are very handy but unfortunately they're not totally bullet-proof.

**Comment by lognaturel** [comment]
No worries, @cdwuestner, this is why we have quick win issues. They're a great opportunity to get used to all the processes in the project.

File: collect_app/src/main/res/xml/analytics_tracker.xml (hunk 1 above)
Comment (cdwuestner) [reply collect_app/src/main/res/xml/analytics_tracker.xml]: Ok, I'll add it back in now.

**Comment by cdwuestner** [comment]
Thanks for both of your help, @lognaturel and @shobhitagarwal1612

**Comment by lognaturel** [comment]
Congrats on your first merge, @cdwuestner! 

The dev community hangs out in the #collect-code channel of https://slack.opendatakit.org and the extended ODK community is on Discourse. You can introduce yourself [here](https://forum.opendatakit.org/t/introduce-yourself-here/6671/62)!

**Pull request merged**